eth-contracts/migrations/3_governance_migration.js

```diff
@@ -19,6 +19,7 @@ const VotingQuorumPercent = 10
 const MaxInProgressProposals = 100
 
 const MaxDescriptionLengthBytes = 250
+const MaxNameLengthBytes = 250
```

Comment:
finalize desired value for `MaxDescriptionLengthInBytes` and `MaxNameLengthInBytes` - while we can potentially remove the max description length, there is still a restriction for maximum string size emitted in events

Reply:
@roneilr @SidSethi if you have input on these final values

Reply:
removing both per offline discussion